airbyte-webapp/src/components/base/Text/Text.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+import React, { useMemo } from "react";
+
+import styles from "./Text.module.scss";
+
+type TextSize = "sm" | "md" | "lg" | "xl";
+type TextElementType = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "span" | "div";
+
+interface TextProps {
+  as?: TextElementType;
+  size?: TextSize;
+  className?: string;
+  bold?: boolean;
+  centered?: boolean;
+}
+
+const getSizeClassName = (size: TextSize): string | undefined => {
+  switch (size) {
+    case "sm":
+      return styles.sm;
+    case "lg":
+      return styles.lg;
+    case "xl":
+      return styles.xl;
+  }
+
+  return undefined;
+};
+
+export const Text: React.FC<TextProps> = ({
+  as = "span",
```

Comment:
Do we have any data on what our default text container is? Should it be `<p />`?

Reply:
That's a good question I wanted to ask! I'm not sure what the default text style should be.